spec/lib/authorities/linked_data/find_term_spec.rb

```diff
@@ -90,90 +90,131 @@
             .to include('Cornell University', 'Ithaca (N.Y.). Cornell University', "Kornel\\xCA\\xB9skii universitet",
                         "K\\xCA\\xBBang-nai-erh ta hs\\xC3\\xBCeh")
         end
+
+        context "ID in graph doesn't match ID in request URI" do
+          before do
```

Comment:
Tests for DataNormalizationError with the OCLC_FAST authority.  Catches the case where special processing of the ID is NOT done.